src/app/shared/register-github-app/register-github-app.component.html

```diff
@@ -33,7 +33,7 @@
       mat-raised-button
       type="button"
       class="accent-1-dark-btn mat-elevation-z no-underline"
-      [href]="gitHubAppInstallationLink$ | async"
+      (click)="registerGitHubApp(gitHubAppInstallationLink$)"
```

Comment:
It appears that `registerGitHubApp` just opens the link in a new tab. This could be done in the HTML file by keeping `[href]="gitHubAppInstallationLink$ | async"` and adding `target="_blank" rel="noopener noreferrer"`